packages/mux-player/src/index.ts

```diff
@@ -683,6 +681,41 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
     this.setAttribute(PlayerAttributes.THEME, `${val}`);
   }
 
+  /**
+   * Get the theme attributes in a plain object (camelCase keys).
+   * This includes already defined attributes. e.g. streamType, disabled, etc.
+   */
+  get themeProps() {
+    const theme = this.mediaTheme;
+    if (!theme) return;
+
+    const props: Record<string, any> = {};
+
+    for (const name of theme.getAttributeNames()) {
+      const value: string | boolean | null = theme.getAttribute(name);
+      props[camelCase(name)] = value === '' ? true : value;
+    }
+
+    return props;
+  }
+
+  /**
+   * Set the theme attributes via a plain object.
+   */
+  set themeProps(props) {
+    this.#init();
+
+    for (const name in props) {
+      const value: string | boolean | null | undefined = props[name];
+
+      if (typeof value === 'boolean' || value == null) {
+        this.mediaTheme?.toggleAttribute(kebabCase(name), Boolean(value));
```

Comment:
`toggleAttribute` can simplify so much of our attribute usage, especially with the forcing parameter. 👍🏻